src/options/options.js

```diff
@@ -10,13 +10,15 @@ import * as RandomTips from "/common/modules/RandomTips/RandomTips.js";
 import * as AddonSettings from "/common/modules/AddonSettings/AddonSettings.js";
 import * as AutomaticSettings from "/common/modules/AutomaticSettings/AutomaticSettings.js";
 import * as CustomOptionTriggers from "./modules/CustomOptionTriggers.js";
-
+import * as ColorSchemeModeHelper from "./modules/ColorSchemeModeHelper.js";
 // init modules
 CustomOptionTriggers.registerTrigger();
 AutomaticSettings.setDefaultOptionProvider(AddonSettings.getDefaultValue);
 AutomaticSettings.init();
+ColorSchemeModeHelper.changeScreenshotTheme();
```

Comment:
nah… rather call it simply `init` here and structure the `ColorSchemeModeHelper` better to have a private function that does this internal stuff. Again, look at the other modules.